macros/edr/data_monitoring/anomaly_detection/construct_metric_properties_dict.sql

```diff
@@ -0,0 +1,7 @@
+{% macro construct_metric_properties_dict(timestamp_column, where_expression, time_bucket) %}
+
+  {%set d = {'timestamp_column': ( timestamp_column if timestamp_column else None ) ,
```

Comment:
Syntax sugar:
```py
timestamp_column or None, # Same as timestamp_column if timestamp_column else None.
where_expression or None
...
```

Reply:
Decided to not change ( I like the trinary expression better than "x or None" )